src/app/app.module.ts

```diff
@@ -6,12 +6,16 @@ import { AppComponent } from './app.component';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { HttpClientModule } from '@angular/common/http';
 import {
+  MatButtonModule,
+  MatExpansionModule,
   MatIconModule,
   MatListModule,
   MatSidenavModule,
+  MatTableModule,
   MatToolbarModule,
+  MatTooltipModule,
 } from '@angular/material';
-import { MatExpansionModule } from '@angular/material/expansion';
+import { ChartsModule } from 'ng2-charts';
```

Comment:
Looks like we're not following import sorting rules?  https://eslint.org/docs/2.0.0/rules/sort-imports